src/03-control-flow.js

```diff
@@ -28,7 +34,14 @@ function isEqual(a, b) {}
  *  findLarger(19, 7);
  *  //> 19
  */
-function findLarger(a, b) {}
+function findLarger(a, b) {
+ if (a > b){
+   return a;
+ }
+ else {
```

Comment:
Remember in cases like this we actually can remove the ` else ` block in favor of a simple return statement.  If a is not greater than b we can simply ` return  b `